actionlist/app/components/action_item.js

```diff
@@ -76,6 +76,16 @@ const mapDispatchToProps = dispatch => ({
 export default connect(null, mapDispatchToProps)(ActionItem);
 
 ActionItem.propTypes = {
-  rowData: React.PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+  rowData: React.PropTypes.shape({
+    rowData: {
```

Comment:
I don't think this is quite right. `rowData` has a prop called `rowData` in it? That doesn't look like how it's used in the component.